README.md

```diff
@@ -1,6 +1,6 @@
 # Schema
 
-## Version 0.12
+## Version 0.12.2-SNAPSHOT
```

Comment:
Should this be the latest published release version?  In this case, that would mean 0.12.1.  We can check the release versions [in the maven repo](https://repo.boundlessgeo.com/release/com/boundlessgeo/schema/)